wildfly-users-ejb/src/main/java/gallb/wildfly/users/ejb/userManagement/business/RoleBean.java

```diff
@@ -23,59 +24,59 @@
 	private Logger oLogger = Logger.getLogger(RoleBean.class);
 
 	@Override
-	public List<Role> getAll() {
+	public List<Role> getAll() throws EjbException {
 		try {
 			@SuppressWarnings("unchecked")
 			List<Role> roles = (List<Role>) oEntityManager.createNamedQuery("Role.findAll").getResultList();
 			return roles;
 		} catch (PersistenceException e) {
 			oLogger.error(e);
-			EjbExeption.getCause(e);
-			throw new EjbExeption(e.getMessage());
+			EjbException.getCause(e);

```

Comment:
EjbException.getCause(e) is dob egy exception-t es a kovetkezo sorba is ujra jobva van egy new EjbException ?